examples/step-37/doc/results.dox

```diff
@@ -556,22 +559,25 @@ values as follows, assuming the Dirichlet values have been interpolated into
 the object @p constraints:
 @code
 template <int dim>
-void LaplaceProblem<dim>::assemble_rhs ()
+void LaplaceProblem<dim>::assemble_rhs()
 {
   solution = 0;
   constraints.distribute(solution);
+  solution.update_ghost_values();
   system_rhs = 0;
 
-  solution.update_ghost_values();
-  FEEvaluation<dim,degree_finite_element> phi(*system_matrix.get_matrix_free());
-  for (unsigned int cell=0; cell<system_matrix.get_matrix_free()->n_macro_cells(); ++cell)
+  const Table<2, VectorizedArray<double>> &coefficient = system_matrix.get_coefficient();
+  FEEvaluation<dim, degree_finite_element> phi(*system_matrix.get_matrix_free());
+  for (unsigned int cell = 0;
+       cell < system_matrix.get_matrix_free()->n_macro_cells();
+       ++cell)
     {
       phi.reinit(cell);
       phi.read_dof_values_plain(solution);
-      phi.evaluate(false, true);
-      for (unsigned int q=0; q<phi.n_q_points; ++q)
+      phi.evaluate(true, true);
```

Comment:
you don't need to change this to `true` here as you only get `phi.get_gradient()`, so it does not need to know how to get values.

Reply:
You're right. I fixed it.